sql/src/test/java/io/crate/operation/reference/sys/check/cluster/LuceneVersionChecksTest.java

```diff
@@ -23,13 +23,34 @@
 package io.crate.operation.reference.sys.check.cluster;
 
 import io.crate.test.integration.CrateUnitTest;
+import org.elasticsearch.Version;
+import org.elasticsearch.cluster.metadata.IndexMetaData;
+import org.elasticsearch.common.settings.Settings;
 import org.junit.Test;
 
 import static org.hamcrest.Matchers.is;
 
 public class LuceneVersionChecksTest extends CrateUnitTest {
 
     @Test
+    public void testRecreationRequired() {
```

Comment:
As before this has nothing to do with lucene, maybe create a TablesNeedRecreationSysCheckTest?